BridgeAppSDK/String+Utilities.swift

```diff
@@ -48,4 +48,20 @@ extension String {
         return self.substringFromIndex(range.endIndex)
     }
     
+    public func stringByRemovingNewlineCharacters() -> String {
+        guard let result = (self as NSString).mutableCopy() as? NSMutableString else { return self }
+        let set = NSCharacterSet.newlineCharacterSet()
+        var range = result.rangeOfCharacterFromSet(set)
```

Comment:
can you do the thing where you split the string on members of the newline character set and then re-join the resulting array with " "? E.g. in obj-C it would be: 
result = [[self componentsSeparatedByCharactersInSet:set] componentsJoinedByString:@" "];
so swift would maybe be something like
self.componentsSeparatedByCharactersInSet(set).componentsJoinedByString(" ")
...?